core/src/main/scala/scalaz/NonEmptyList.scala

```diff
@@ -131,7 +131,7 @@ object NonEmptyList extends NonEmptyListInstances {
   def apply[A](h: A, t: A*): NonEmptyList[A] =
     nels(h, t: _*)
 
-  def unapplySeq[A](v: NonEmptyList[A]): Option[(A, IList[A])] =
+  def unapply[A](v: NonEmptyList[A]): Option[(A, IList[A])] =
```

Comment:
`unapplySeq` requires that the second argument be a `Seq`, which `IList` is not. The old one was totally unusable. I had a choice between changing to unapply and using the `List` form for pattern matching. This is cheaper and actually usable.